agcaballero/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -47,6 +50,42 @@ boolean isReplaceableBy(Availability availability) {
    
   /** the number of minutes in a day */
   private static final int MINUTES_IN_DAY = 24 * 60;
+
+  /**
+   * A private class that holds the necessary elements of availability:
+   * can all mandatory attendees attend and how many optional attendees can't come.
+   */
+  private class Availability {
```

Comment:
It's a bit confusing to have this class named the same as the enum above (I'm surprised this is allowed). For example, it could be hard to figure out which one List<Availability> contains. Can the two be merged into the enum, or can they be named differently?

Reply:
I will rename the enum to AvailabilityOptions. Thank you for the suggestion!

Reply:
Done